modules/@apostrophecms/doc/index.js

```diff
@@ -129,6 +129,9 @@ module.exports = {
             }
           });
           doc.updatedAt = new Date();
+          doc.updatedBy = req.user || {
```

Comment:
You should cherrypick the likely suspects: firstName, lastName, title, and _id. Although passwords are not in req.user restraint in exposing it as a whole is a good practice. Otherwise A-OK